extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -60,9 +62,46 @@ require([
 			inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
 			inNextVideoAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoNextVideoAutoplayCountries),
 			autoplayCookieName = 'featuredVideoAutoplay',
-			autoplay = cookies.get(autoplayCookieName) !== '0' && inAutoplayCountries;
+			autoplay = cookies.get(autoplayCookieName) !== '0' && inAutoplayCountries,
+			context = adContext && adContext.getContext(),
+			logGroup = 'FeaturedVideo';
 
 		function initVideo(onCreate) {
+			var adNotBlockedCallback = function () {
```

Comment:
I'd add a comment or name this function as `detectAdBlock` so someone looking at this code will know what it does at first glance